knowledge/musical_instruments/les_paul_studio/qna.yaml

```diff
@@ -0,0 +1,27 @@
+created_by: 
```

Comment:
Please add your GitHub Username here.

Reply:
Please add another commit with that change. Then we can run the linter to check the file(s).

Reply:
I did this during Summit with on premise support and instruction. Not a coder by trade, could you advise

Reply:
Ah ok. 

There are some standard way to add and update PRs, and me walking you through this is unrealistic through comments on the PR. Please google "standard Pull request GitHub" or the like and attempt to update this PR. 

If you don't want to continue that's fully understandable, and please close this PR below so we can remove it from our queue